data/schemas/keyboard.jsonschema

```diff
@@ -504,5 +513,46 @@
                 "speaker": {"$ref": "qmk.definitions.v1#/mcu_pin"}
             }
         }
+        "serial": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "driver": {
+                    "type": "string",
+                    "enum": ["bitbang", "vendor", "usart"]
+                },
+                "full_duplex": {"type": "boolean"}
+                "tx_pin": {"$ref": "qmk.definitions.v1#/mcu_pin"},
+                "rx_pin": {"$ref": "qmk.definitions.v1#/mcu_pin"},
+                "debug": {"type": "boolean"}
+            }
+        },
+        "eeprom": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "driver": {
+                    "type": "string",
+                    "enum": ["vendor", "custom", "transient", "i2c", "spi", "wear_leveling"]
+                },
+                "wear_leveling": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "driver": {
+                            "type": "string",
+                            "enum": ["custom", "embedded_flash", "spi_flash", "rp2040_flash", "legacy"]
+                        },
+                    }
+                },
+            }
+        },
+        "via": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "enabled": {"type": "boolean"},
+            }
+        },
```

Comment:
in theory this should already be covered by:
```json
{
    "features": {
        "via": true
    }
}```

Reply:
where is this from?
i can't find that in develop, the only occurrence there of `"via": true` is in a single keyboard's keymap.json

Reply:
<https://github.com/qmk/qmk_firmware/blob/master/lib/python/qmk/info.py#L122>

Reply:
jup, that works